curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-92-square-digit-chains.md

```diff
@@ -15,41 +15,141 @@ For example,
 <div style='margin-left: 4em;'>
   44 → 32 → 13 → 10 → <strong>1</strong> → <strong>1</strong><br>
   85 → <strong>89</strong> → 145 → 42 → 20 → 4 → 16 → 37 → 58 → <strong>89</strong>
-</div>
+</div><br>
```

Comment:
Can this bit be better represented with KaTeX?

Reply:
Yes! Might be even nicer than expected:
![image](https://user-images.githubusercontent.com/60067306/119446858-5dd0a700-bd2f-11eb-8922-3ae19f5dac2f.png)
What do you think?

```suggestion
$$\begin{align}
  & 44 → 32 → 13 → 10 → \boldsymbol{1} → \boldsymbol{1}\\\\
  & 85 → \boldsymbol{89} → 145 → 42 → 20 → 4 → 16 → 37 → 58 → \boldsymbol{89}\\\\
\end{align}$$
```

Reply:
Yep I think that's the way to go.